services/py-api/src/server/handlers/participants_handlers.py

```diff
@@ -1,3 +1,4 @@
+from fastapi.responses import JSONResponse
```

Comment:
You won't need this import. See the comment below.

Reply:
Fixed in 1bec98abe406eea6382a5453ac98f3e900d3ddd4